Source/Scene/GlobeDepth.js

```diff
@@ -51,19 +56,35 @@ function GlobeDepth() {
 }
 
 Object.defineProperties(GlobeDepth.prototype, {
+  colorFramebufferManager: {
```

Comment:
Between this getter and `framebuffer`, maybe the object properties are getting a bit unclear. In general I was trying to make the `FramebufferManager` class be invisible by not referring to things that used to be framebuffers by a different name, but in this case I think it makes more sense. Definitely open to changing this if there are other suggestions.